_pages/integrations/arize.md

```diff
@@ -0,0 +1,253 @@
+
+{% highlight python %}
+import os
+import uuid
+
+import Algorithmia
+from arize.api import Client
+from arize.types import ModelTypes
+import joblib
+import pandas as pd
+import shap
+
+
+# Define where model and sample data live.
+COLLECTION_OWNER = "COLLECTION_OWNER"
+COLLECTION_NAME = "COLLECTION_NAME"
+MODEL_NAME = "Algorithmia_Tutorial_Model.joblib"
+
+# Load keys from environment variables and instantiate clients.
+ARIZE_API_KEY = os.getenv("ARIZE_API_KEY")
+ARIZE_ORG_KEY = os.getenv("ARIZE_ORG_KEY')
+ALGORITHMIA_API_KEY = os.getenv("ALGORITHMIA_API_KEY")
+
+arize_client = Client(organization_key=ARIZE_ORG_KEY, api_key=ARIZE_API_KEY)
+algo_client = Algorithmia.client(ALGORITHMIA_API_KEY, CLUSTER_DOMAIN)
+
+
+def load_model(data_uri):
+    """Load sklearn model from data collection and return model object."""
+    model_path = algo_client.file(data_uri).getFile().name
+    model = joblib.load(model_path)
+    return model
+
+# Load model.
+model_file_path = "data://"+COLLECTION_OWNER+"/"+COLLECTION_NAME+"/"+MODEL_NAME 
+clf = load_model(model_file_path)
+
+
+def apply(input):
+    # Read JSON input to pd.DataFrame.
+    X_data = pd.read_json(input)
+    # Generate new predictions in production.
+    y_pred = clf.predict(X_data)
+
+    shap_values = shap.Explainer(clf, X_data).shap_values(X_data)
+    shap_values = pd.DataFrame(shap_values, columns=X_data.columns)
+
+    ids = pd.Series([str(uuid.uuid4()) for _ in range(len(X_data))])
+
+    # Log the prediction data to Arize.
+    log_responses = arize_client.log_bulk_predictions(
+        model_id="Algorithmia_Tutorial_Model",
+        model_version="1.0",
+        model_type=ModelTypes.BINARY,
+        features=X_data,
+        prediction_ids=ids,
+        prediction_labels=pd.Series(y_pred))
+
+    # Log the SHAP data to Arize.
+    shap_responses = arize_client.log_bulk_shap_values(
+        model_id="Algorithmia_Tutorial_Model",
+        # Pass in the same prediction IDs to match predictions and actuals.
+        prediction_ids=ids,
+        shap_values=shap_values
+    )
+
+    # Return prediction data.
+    res = pd.DataFrame(y_pred)
+    res.index = ids
+    res.index.rename("pred_id", inplace=True)
+    return res.to_json()
+{% endhighlight %}
+
+The following code is intended to be executed back in the same external environment (Jupyter notebook or external training platform) that you used above to train your algorithm, once you've built the algorithm on Algorithmia. {:.notice-info}
+
+Once you've built your algorithm, you can call it using its version hash to test it out; this will be a value like `f35025657bdc37eb0d6ffeed62b0539ee21c8b4e`. If you build your algorithm in the browser IDE, this hash is displayed in the test console output upon successful build completion, but it's also available in the "Builds" tab on the algorithm's homepage. You can also publish the algorithm, in which case you'll be able to call the algorithm using a semantic version such as `1.0.0`.
```

Comment:
```suggestion
Once you've built your algorithm, you can call it using its version hash to test it out; this will be a value like `f35025657bdc37eb0d6ffeed62b0539ee21c8b4e`. If you build your algorithm in the Algorithmia Web IDE, this hash is displayed in the test console output upon successful build completion, but it's also available in the "Builds" tab on the algorithm's homepage. You can also publish the algorithm, in which case you'll be able to call the algorithm using a semantic version such as `1.0.0`.
```